source/common/network/connection_impl.cc

```diff
@@ -531,11 +531,12 @@ void ConnectionImpl::updateWriteBufferStats(uint64_t num_written, uint64_t new_s
                                            connection_stats_->write_current_);
 }
 
-ClientConnectionImpl::ClientConnectionImpl(Event::Dispatcher& dispatcher,
-                                           const Address::InstanceConstSharedPtr& remote_address,
-                                           const Address::InstanceConstSharedPtr& source_address)
-    : ConnectionImpl(dispatcher, remote_address->socket(Address::SocketType::Stream),
-                     remote_address, nullptr, source_address, false, false) {}
+ClientConnectionImpl::ClientConnectionImpl(
+    Event::Dispatcher& dispatcher, const Address::InstanceConstSharedPtr& address,
```

Comment:
nit: can we retain the naming of remte_address/source_address rather than address?

Reply:
Done